ara/plugins/callback/ara_default.py

```diff
@@ -521,6 +521,11 @@ def _load_result(self, result, status, **kwargs):
         # Retrieve the host so we can associate the result to the host id
         host = self._get_or_create_host(hostname)
 
+        # If the task was delegated to another host, retrieve that too
+        delegated_to = None
+        if result._task.delegate_to and result._task.delegate_to != result._host.get_name():
+            delegated_to = self._get_or_create_host(result._task.delegate_to)
```

Comment:
It turns out that ``result._task.delegate_to`` is not "resolved" if the task is a loop and can return a jinja expression instead:
![Screenshot from 2021-07-27 14-52-55](https://user-images.githubusercontent.com/1291204/127211360-e16e1fa8-e3d2-46c1-9de6-ed215f8cb67a.png)

Talked to #ansible-devel about it and it's a bug they can fix but we need to find a workaround in the meantime.

Reply:
Upstream issue: https://github.com/ansible/ansible/issues/75339

Reply:
Found a workaround and sent a PR to fix it: https://github.com/ansible-community/ara/pull/306